lang/py/src/avro/ipc.py

```diff
@@ -296,9 +296,9 @@ def respond(self, call_request):
       # perform server logic
       try:
         response = self.invoke(local_message, request)
-      except AvroRemoteException, e:
+      except AvroRemotelxception as e:
```

Comment:
Definitely no Python guru here, so don't consider this an actual review, but this is most likely a typo: `AvroRemotelxception`.

Reply:
Thanks didn't catch that one, Ill update my pull request